spec/models/complete_moab_spec.rb

```diff
@@ -300,14 +311,14 @@
     describe '.by_druid' do
       it 'returns the expected complete moabs' do
         expect(described_class.by_druid(druid).length).to eq 1
-        expect(described_class.by_druid('bj102hs9687')).to be_empty
+        expect(described_class.by_druid('bj102hs9687')).to be_empty # bj102hs9687 from preserved_object factory
       end
     end
 
     describe '.by_storage_root' do
       it 'returns the expected complete moab when chained with by_druid' do
+        expect(described_class.by_druid(druid).length).to eq 1
         expect(described_class.by_druid(druid).by_storage_root(cm.moab_storage_root).length).to eq 1
-        expect(described_class.by_druid(druid).by_storage_root(MoabStorageRoot.first)).to be_empty
```

Comment:
as John and I discussed, this expectation doesn't map to a non-multi moab world.